src/Uno.UI/UI/Xaml/DependencyObjectStore.cs

```diff
@@ -1129,7 +1129,7 @@ internal void UpdateResourceBindings(bool isThemeChangedUpdate, ResourceDictiona
 					if (dict.TryGetValue(kvp.Value.ResourceKey, out var value, shouldCheckSystem: false))
 					{
 						wasSet = true;
-						SetValue(kvp.Key, BindingPropertyHelper.Convert(() => kvp.Key.Type, value));
+						SetValue(kvp.Key, BindingPropertyHelper.Convert(() => kvp.Key.Type, value), kvp.Value.Precedence);
```

Comment:
I forgot to ask in the previous PR, are there ways to validate that in runtime/unit test ?

Reply:
depends, do you mean Uno.UI.RuntimeTests? It should be possible to run the tests there, now that `SwapSystemTheme()` can be implemented (or faked) by setting Window.Current.Content.RequestedTheme